### PR TITLE
Add quit option to app menu and open app from tray upon left click

### PR DESCRIPTION
**Pull request by danielhusar**
Since app by default go to tray I think its nice to have quit option in file dropdown.
Also left click on tray does nothing and most users probably expect app to open.

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -189,6 +189,15 @@ const linuxTpl = [
 				click() {
 					sendAction('log-out');
 				}
+			},
+			{
+				type: 'separator'
+			},
+			{
+				label: `Quit ${appName}`,
```
Comment (sindresorhus) [inline menu.js]: Should just be `Quit`.

**Comment by kevva** [comment]
:+1:

File: menu.js (hunk 1 above)
Comment (danielhusar) [inline menu.js]: Sure but you have it same in OSX :) (altrough its not under File dropdown)
https://github.com/danielhusar/caprine/blob/improvements/menu.js#L75

Comment (sindresorhus) [inline menu.js]: The convention differ between OSes. On Linux (Ubuntu) it's mostly `Quit`.

Comment (kevva) [inline menu.js]: It's `Quit ${appName}` on OS X. But maybe the naming convention differs?

Comment (danielhusar) [inline menu.js]: :+1:

**Comment by kevva** [comment]
Nice work, thanks!

**Pull request merged**